force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -253,12 +293,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer sumPositiveIntegers() {
         List<Integer> numbers = new List<Integer>{-1, 2, -3, 4, -5, 6}; //DO NOT CHANGE
-
+        Integer sum = 0;
         // Loop through the list of integers
             // if the number is negative skip this iteration
-        
-
-        return null; // Replace null with the variable you used to store the result
+        for(Integer num : numbers){
+            if(num < 0){
+                continue;
+            }
+            sum = sum + num;
+        }
+        return sum; // Replace null with the variable you used to store the result
```

Comment:
Good work here. My suggestion is that you don't need the continue block. If num > 0 then do your thing. Otherwise the loop will continue to the next member regardless.